homeassistant/components/pi_hole/__init__.py

```diff
@@ -1 +1,96 @@
 """The pi_hole component."""
+import logging
+
+import voluptuous as vol
+
+from hole import Hole
```

Comment:
Please add a blank line between 3rd party and homeassistant import groups. `hole` should be grouped with `voluptuous`.